README.md

```diff
@@ -1,9 +1,243 @@
 # ros_utilities
 Wrappers and other utilities for ROS2
 
+## Getting started
+
+### Process-wide APIs
+
+These APIs wrap [`rclpy`](https://github.com/ros2/rclpy) APIs to provide a simpler UX in common use cases.
+All APIs are thread-safe and exception-safe. Resource management is automatic. By default, callbacks are
+dispatched in the background, enabling both synchronous and asynchronous programming.
+
+These APIs are also quite handy to reconcile past [`rospy`](http://wiki.ros.org/rospy) experience with ROS 2.
```

Comment:
I feel like a little bit more conceptual stuff here might help with understanding the examples.  I'd throw in some high level stuff about how the backend works.  For instance, a "process-wide node" is referenced a few times - I think it'd be worth saying what that is.

Reply:
Good idea. Done in https://github.com/bdaiinstitute/ros_utilities/pull/32/commits/efb0db0d1ef57bdecc49aaff0dcbe9222defb1e3.